pages/index.js

```diff
@@ -13,6 +61,32 @@ export default function Home() {
             <SpotCard spot={spot} key={spot.name}></SpotCard>
           ))}
       </SpotList>
+
+      <Button
+        type="button"
+        variant="create"
+        onClick={() => {
+          setState((prevState) => {
+            if (prevState === "hide") {
+              return "show";
+            } else {
+              return "hide";
+            }
+          });
+        }}
+      >
+        Parenthood Spot hinzufügen
+      </Button>
+      {state === "show" ? (
+        <>
+          <CreateSpot
+            closeCreatePage={closeCreatePage}
+            addSpot={addSpot}
+            state={state}
+            spot={spots}
+          ></CreateSpot>
+        </>
+      ) : null}
```

Comment:
A refactored more readable solution in combination withe state which hold as boolean as a value. 
👉**Super-Tipp** [Conditional Rendering with Logical && Operator](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator)

```suggestion
  <Button
        type='button'
        variant='create'
        onClick={() => {
          setIsShown((prevState) => !prevState);
        }}>
        Parenthood Spot hinzufügen
      </Button>
      {isShown && (
        <CreateSpotCard
          closeCreatePage={closeCreatePage}
          addSpot={addSpot}
          isShown={isShown}
          spot={spots}></CreateSpotCard>
      )}
```